alluxio/alluxio.rst

```diff
@@ -1,3 +1,13 @@
 =========
 Alluxio
 =========
+
+|fdd| adopts Alluxio |alluxio-version| as the default filesystem
+
```

Comment:
Perhaps it would be beneficial to have a link to the official docs here as well.